### DISCUSSION TITLE
Getting an IPFS peer to start Bacalhau Server

### DISCUSSION THREAD

**Discussion by fakela** [bacalhau-project/bacalhau#1689] [Q&A] [answered]
How do I get an IPFS peer if I want to start Bacalhau Server?

**Comment by enricorotundo** [comment, marked as answer]
A viable (but not necessarily the easiest 😓) option is to run your own IPFS daemon and fetch your multiaddress as explained [here](https://docs.bacalhau.org/running-node/quick-start#ensure-ipfs-is-running).
